.travis.yml

```diff
@@ -1,26 +1,23 @@
 sudo: false
 
 language: ruby
-
 cache: bundler
-
 rvm:
   - 2.0.0
   - 2.1
   - 2.2.6
   - 2.3.3
-
+  - 2.4.0
```

Comment:
I'd prefer to have this in a separate PR (we already have #45 for this) so we can revert things separately if necessary. It also will make clearer what commit changed what.